user_management_app/utils.py

```diff
@@ -0,0 +1,25 @@
+from django.http import HttpResponse
+
+
+def get_user_not_found_response():
+    response = HttpResponse("User doesn't exist")
+    response.status_code = 401
+    return response
+
+
+def get_unauthorized_operation_response():
+    response = HttpResponse("Operation not allowed: Unauthorized user")
+    response.status_code = 401
```

Comment:
REST framework provides more explicit identifiers for each status code, such as HTTP_400_BAD_REQUEST in the status module. It's a good idea to use these throughout rather than using numeric identifiers. http://www.django-rest-framework.org/tutorial/2-requests-and-responses/#response-objects